docs/BEMAN_STANDARD.md

```diff
@@ -422,3 +422,21 @@ copyright notice following the SPDX license identifier.
 
 **[CPP.NAMESPACE]** RECOMMENDATION: Headers in `include/beman/<short_name>/` should export
 entities in the `beman::<short_name>` namespace.
+
+## Continous Integration
+
+**[CI.PLATFORM]** REQUIREMENT: All CI should be implemented using GitHub Actions or an CI platform that appripriatly interacts with the code control system.
```

Comment:
`an CI platform that appropriately interacts with the code control system` may need better wording here.

Reply:
If someone wants to pay for an external CI system, that might be acceptable? We can cross that bridge if and when. 

But for beman standards purposes, configuration as code is important, and that code should live with the project. That's built in with GH Actions, but it does mean I can mostly coax Gitea to use them, too. I haven't tried Gitlab. 

Internally, we use GHE, but not Actions, and instead use our own systems for CI and CD, but they're sufficiently agnostic that it's not an issue. It is why I prefer as minimal magic in the actions as possible, though, so it's straightforward to run tests on change.

Reply:
Yeah I agree, we need to come up with a wording for allowing acceptable external CI or make this item a recommendation.